src/UI/Components/Diagram/shapes.ts

```diff
@@ -254,13 +255,13 @@ export class ServiceEntityBlock extends shapes.standard.HeaderedRecord {
       },
     );
 
-    this.set("entityName", name);
-    this.attr(["headerLabel", "data-testid"], "header-" + name);
+    this.set("entityName", name); //name is still assigned to the entityName attribute, not to disturb the existing logic
```

Comment:
hmm, might be more confusing later on tho. No?

Reply:
I thought about this, and I think it's better to keep it as it is, because the data that matters in case of the logic is the name. we want to display the type but it is actually not relevant. I can add more to the comment if you want.

Reply:
yeah that's okay, a bit more context never hurts